assets/helpers/tracking/acquisitions.js

```diff
@@ -246,6 +247,7 @@ function derivePaymentApiAcquisitionData(
     componentType: referrerAcquisitionData.componentType,
     source: referrerAcquisitionData.source,
     abTests,
+    gaId: getCookie('_ga'),
```

Comment:
Is this getting sent to the payment api and ultimately passed into the acquisition-event-producer library?

Reply:
yes, see https://github.com/guardian/payment-api/pull/130/files